chain/client.go

```diff
@@ -44,7 +44,7 @@ func NewChainClient(ccc *lens.ChainClientConfig, input io.Reader, output io.Writ
 		Config:         ccc,
 		Input:          input,
 		Output:         output,
-		Codec:          lens.MakeCodec(ccc.Modules),
+		Codec:          lens.MakeCodec(ccc.Modules, []string{}),
```

Comment:
Do we need to add "ethermint" as an extra codec?
Before lens upgrade https://github.com/VolumeFi/lens-1/blob/v0.5.1/client/encoding.go
The `MakeCodec` func registers the interfaces to the ethermint codec by default.
But in the upgrade to sdk 0.47 https://github.com/VolumeFi/lens-1/blob/def8a6f4cbfdc30e39dd70b6f364422ef1f7c262/client/encoding.go#L31
We are required to add ethermint as an extra codec.
@ToasterTheBrave @taariq @aleem1314